apollo-router/src/plugins/authorization/mod.rs

```diff
@@ -107,6 +120,17 @@ impl AuthorizationPlugin {
         Ok(has_config.unwrap_or(true) && has_authorization_directives)
     }
 
+    pub(crate) fn log_errors(configuration: &Configuration) -> bool {
+        let has_config = configuration
+            .apollo_plugins
+            .plugins
+            .iter()
+            .find(|(s, _)| s.as_str() == "authorization")
+            .and_then(|(_, v)| v.get("preview_directives").and_then(|v| v.as_object()))
+            .and_then(|v| v.get("log_errors").and_then(|v| v.as_bool()));
+        has_config.unwrap_or(true)
+    }
```

Comment:
I know that looks useless but could you write a small test to check that function ? Basically it's to prevent breaking change when we will remove `preview_` prefix and if we decide to change the configuration shape later